src/test_l1_dsVideoDevice.c

```diff
@@ -1651,6 +1834,10 @@ int test_l1_dsVideoDevice_register ( void )
     UT_add_test( pSuite2, "dsForceDisableHDRSupport_L1_positive" ,test_l1_dsVideoDevice_positive_dsForceDisableHDRSupport );
     UT_add_test( pSuite2, "dsGetDFC_L1_negative" ,test_l1_dsVideoDevice_negative_dsGetDFC );
     UT_add_test( pSuite2, "dsForceDisableHDRSupport_L1_negative" ,test_l1_dsVideoDevice_negative_dsForceDisableHDRSupport );
+    UT_add_test( pSuite2, "dsRegisterFrameratePreChangeCB_L1_positive" ,test_l1_dsVideoDevice_positive_dsRegisterFrameratePreChangeCB );
+    UT_add_test( pSuite2, "dsRegisterFrameratePreChangeCB_L1_negative" ,test_l1_dsVideoDevice_negative_dsRegisterFrameratePreChangeCB );
```

Comment:
The tests are added to pSuite2, which is [L1 dsVideoDevice - advanced]

Can you add it to pSuite1 ?

Reply:
Done